docs/get-started/terminology.md

```diff
@@ -10,3 +10,6 @@ This document provides clear definitions of key concepts used in **Concrete** fr
 
 * **Circuit:** The result of compilation. A circuit includes both client and server components. It has methods for various operations, such as printing and evaluation.
 
+* **Table lookup (TLU):** Table loopup (TLU) stands for instructions of the form `y = T[i]`. In practice, in FHE, they are done with programmable bootstrapping, which is the equivalent operation on encrypted values.
+
+* **Programmable bootstrapping (PBS):** Programmable bootstrapping (PBS) are the equivalent of table lookup `y = T[i]` on encrypted values, i.e., where input `i` and output `y` are encrypted (but not the table `T`).
```

Comment:
```suggestion
* **Programmable Bootstrapping (PBS):** PBS is equivalent to looking up the table `y = T[i]` on encrypted values, which means that the inputs `i` and the outputs `y` are encrypted, but the table `T` is not encrypted.
```

Reply:
"looking up the table" looks strange to me to. "PBS is equivalent to table lookup" would be ok for me

Reply:
@yuxizama tell me if we can change this, please

Reply:
done